modules/Hoot/ui/sidebar/advancedOpts.js

```diff
@@ -218,18 +223,47 @@ export default class AdvancedOpts {
             .classed( 'group-toggle-caret-wrap', true)
             .append( 'div' )
             .attr( 'class', 'adv-opt-toggle' )
-            .classed( 'combobox-caret', d => d.members.length )
-            .on( 'click', function(d) {
-                if (d.members.length) {
-                    let body      = d3.select( `#${ d.name }_group` ).select( '.group-body' ),
-                        bodyState = body.classed( 'hidden' );
-
-                    body.classed( 'hidden', !bodyState );
-                    body.classed( 'keyline-bottom', bodyState );
-                }
-            });
+            .classed( 'combobox-caret', d => d.members.length );
 
         caretWrap.merge(caretWrapEnter);
+        caretWrap.on( 'click', instance.showBody);
+    }
+
+    showBody(d) {
+
+        if (d3.event.target.classList.contains( 'conflate-type-toggle' )) return;
```

Comment:
Ignore the showBody function from firing if we click the checkbox.
Otherwise, the toggleOption function gets called twice and the user just experiences their click to be a noop.